src/main/java/io/cdap/plugin/servicenow/source/ServiceNowSourceConfig.java

```diff
@@ -452,4 +453,14 @@ public boolean shouldConnect() {
       !containsMacro(ServiceNowConstants.PROPERTY_USER) &&
       !containsMacro(ServiceNowConstants.PROPERTY_PASSWORD);
   }
+
+  protected boolean shouldGetSchema() {
```

Comment:
Why is this protected? Can we make it public?

Reply:
At the time of writing this method, I was referring to a similar method in a different connector that had `protected` as its access modifier.

Have changed it to `public`
8fbc7cfe28c54cafac8ef18493777c7657cb47e5